assets/js/lib/fetchLocation.js

```diff
@@ -0,0 +1,23 @@
+async function fetchLocationDataFromAPI(){
+    let locationData = []
+    let response = await fetch(cowinMapDataAPIUrl);
+    let jsonData = await response.json();
+    window.sessionStorage.setItem(cachedCowinMapDataKey, JSON.stringify(jsonData));
```

Comment:
I'm not sure if we should cache this data. This will lead to confusion when our teams update the backend resources and don't see that reflect on the front-end. They might not realise that the have to open a new tab. 

If this becomes a problem, we will have to solve with server caching on the backend side. If I remember correctly, back-end already caches this information. We should we fine to call it multiple times.

So, let's remove this caching.

Reply:
I'm caching this because when the user changes their state/city location, there is a huge latency in getting the map data - I know I'm testing from the US & that it's faster in India, but we'd want to keep the user-perceived latency, as minimal as possible. I can add to the READ ME of both backend & frontend informing this behavior - so the teams are aware of this.

Reply:
I think the latency on this request won't have much impact on the user experience as I understand that this request doesn't really block the actual loading of resources that are plotted on the map. 

Also, after speaking to @nmahesh1412 and @chaitanyasama, I also realised that many of our team members are also the volunteers who don't have access to the codebases' documentation. This will make it hard to keep communicate and maintain this knowledge in the team. We will also have a lot of new developers added to the project and even with them, it will be hard to ensure this is well understood despite clear documentation. So, let's remove the caching of this request and let the browser handle the caching through it's default caching behaviour on requests.

Reply:
done